x-pack/plugins/security_solution/public/common/components/exceptions/use_add_exception.tsx

```diff
@@ -139,20 +141,23 @@ export const useAddOrUpdateException = ({
             prepareExceptionItemsForBulkClose(exceptionItemsToAddOrUpdate),
             false
           );
-          await updateAlertStatus({
+
+          const response = await updateAlertStatus({
             query: {
               query: filter,
             },
             status: 'closed',
             signal: abortCtrl.signal,
           });
+          conflicts = response.version_conflicts;
+          updated = response.updated;
         }
 
         await addOrUpdateItems(exceptionItemsToAddOrUpdate);
 
         if (isSubscribed) {
           setIsLoading(false);
-          onSuccess();
+          onSuccess(updated, conflicts);
```

Comment:
Super nit: could this also be shortened a bit to just be:

```suggestion
          onSuccess(response.updated ?? 0, response.version_conflicts ?? 0);
```

Reply:
I tried this, but `response` was out of scope at this point. I did just uncover a related bug here (actually, the unit tests uncovered it)... let me know what you think about the updates. :)